public/app/percona/backup/components/AddBackupModal/AddBackupModal.utils.ts

```diff
@@ -59,11 +59,12 @@ export const toFormBackup = (backup: Backup | ScheduledBackup | null): AddBackup
       startMinute: [{ value: 0, label: '00' }],
       logs: false,
       active: true,
+      vendor: (null as unknown) as Databases,
```

Comment:
Can the type for `vendor` be `string | null`? So there won't be the need to cast to `unknown`

Reply:
Good one 👍